index.html

```diff
@@ -0,0 +1,128 @@
+        <h2>Hand Crafted, home made masterpices</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            name="email"
+            id="email"
+            type="email"
+            placeholder="Enter your email address"
+            required=""
+          />
+          <input id="submit" type="submit" value="Get Started" class="button" />
+        </form>
+      </section>
+      <article class="p-details">
+        <section id="features">
+          <div class="grid">
+            <div class="details">
+              <h2>Premium Materials</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+          <div class="grid">
+            <div class="details">
+              <h2>Fast Shipping</h2>
+              <p>
+                We make sure you recieve your trombone as soon as we have
+                finished making it. We also provide free returns if you are not
+                satisfied.
+              </p>
+            </div>
+          </div>
+          <div class="grid">
+            <div class="details">
+              <h2>Quality Assurance</h2>
+              <p>
+                For every purchase you make, we will ensure there are no damages
+                or faults and we will check and test the pitch of your
+                instrument.
+              </p>
+            </div>
+          </div>
+        </section>
+        <section id="how-it-works">
+          <iframe
+            id="video"
+            height="315"
+            src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0"
+            frameborder="0"
+            allowfullscreen=""
+          ></iframe>
+        </section>
+        <section id="pricing">
+          <div class="products" id="tenor">
+            <div class="p1">Tenor Trombone</div>
+            <h2>$550</h2>
+            <ol>
+              <li>lorem ipsum</li>
+              <li>lorem ipsum</li>
+              <li>lorem ipsum</li>
+            </ol>
+            <button type="button" class="button">Add</button>
+          </div>
+
+          <div class="products" id="bass">
+            <div class="p1">Bass Trombone</div>
+            <h2>$750</h2>
+            <ol>
+              <li>lorem ipsum</li>
+              <li>lorem ipsum</li>
+              <li>lorem ipsum</li>
+            </ol>
+            <button type="button" class="button">Add</button>
+          </div>
+
+          <div class="products" id="valve">
+            <div class="p1">Valve Trombone</div>
+            <h2>$1050</h2>
+            <ol>
+              <li>lorem ipsum</li>
+              <li>lorem ipsum</li>
+              <li>lorem ipsum</li>
+            </ol>
+            <button type="button" class="button">Add</button>
+          </div>
+        </section>

```

Comment:
# Required Changes ♻️

Kindly, I suggest you implement FLEXBOX and other styles in ID "pricing", this detail so that your site looks more like accurate alignment